### ISSUE TITLE
test monitored resource label consistency

### ISSUE THREAD

**Issue by daniel-sanche** [googleapis/env-tests-logging#58]
We have a [test](https://github.com/googleapis/env-tests-logging/blob/f880c708984e05ad8d03fd4785279fa87db009d6/tests/common/common.py#L160) to make sure monitored resources are captured, but it checks labels against a [list specific to each language/environment pair](https://github.com/googleapis/env-tests-logging/blob/f880c708984e05ad8d03fd4785279fa87db009d6/tests/common/common.py#L169). We should define our expectations for each environment, and make sure it is tested consistently across languages

**Comment by minherz** [comment]
It would be great if we can elaborate this FR little more. [Public documentation](https://cloud.google.com/monitoring/api/resources) describes labels expected with each resource type.
The list of the supported resource types is very long, the libraries support only the following:
| resource type | environment | comment |
|----|----|----|
| gce_instance | GCE ||
| gae_app | GAE | Java tries to public suffixes _flex and _standard to reflect on two types of GAE but it is inconsistent with Google Cloud |
| k8s_container | GKE ||
| cloud_run_revision | Cloud Run ||
| cloud_function | Cloud Function | Currently not supported in Java |
| global | Global ||

**Comment by daniel-sanche** [comment]
Sorry, most of these issues were created assuming I'm the one coming back to them later, so I didn't go into much detail. 

The way the [current monitored resource environment test](https://github.com/googleapis/env-tests-logging/blob/f880c708984e05ad8d03fd4785279fa87db009d6/tests/common/common.py#L169) works is it checks to make sure each label in `self.monitored_resource_labels` is set. `self.monitored_resource_labels` is [configured differently for each individual language/environment pair](https://github.com/googleapis/env-tests-logging/blob/f880c708984e05ad8d03fd4785279fa87db009d6/tests/python/test_cloudrun.py#L33). So there is no guarantee the Python library on Cloud Run is populating identical fields to the java library running on the same environment.

Our end goal is to make sure the languages all behave consistently, so we want to make sure the languages are all populating the same fields the same way when run on the same environment.

TL;DR: This issue is about re-writing [this test](https://github.com/googleapis/env-tests-logging/blob/f880c708984e05ad8d03fd4785279fa87db009d6/tests/common/common.py#L160) to be more specific in what it expects

**Comment by daniel-sanche** [comment]
> I would expect from the test to verify that the same set of labels is populated by the libraries when structured logging is enabled.

It's not just when structured logging is enabled, we still want to capture the monitored resources and all the proper metadata when using grpc to send logs
